src/actions/action-types.js

```diff
@@ -1,5 +1,7 @@
 // Event actions
 export const LOAD_EVENTS_LIST_SUCCESS = 'LOAD_EVENTS_LIST_SUCCESS';
+export const SET_EVENT_PROFILE = 'SET_EVENT_PROFILE';
```

Comment:
https://github.com/cloudmu/react-redux-starter-kit/blob/master/src/actions/alerts.js
Check this example. Personally, I would prefer to have actions and action creators related to specific reducer in one file.

Reply:
fixed